packages/gatsby-telemetry/src/store.js

```diff
@@ -0,0 +1,76 @@
+const { homedir } = require(`os`)
+const path = require(`path`)
+const {
+  appendFileSync,
+  readFileSync,
+  renameSync,
+  existsSync,
+  unlinkSync,
+} = require(`fs`)
+const { ensureDirSync } = require(`fs-extra`)
+const Configstore = require(`configstore`)
+
+module.exports = class Store {
+  config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
```

Comment:
We might want to wrap this in a try catch for certain environments when permissions don't allow writing to ~/.config. 

Not blocking and can do later though!

Reply:
Actually, never mind. This should be fine since it falls back to os.tmpdir() if no XDG Base Directory is found

https://github.com/yeoman/configstore/blob/0df1ec950d952b1f0dfb39ce22af8e505dffc71a/index.js#L11